dataactvalidator/config/sqlrules/b11_award_financial_3.sql

```diff
@@ -3,4 +3,6 @@ SELECT
     af.object_class
 FROM award_financial AS af
 WHERE af.submission_id = {}
-AND af.object_class = '000'
+AND LENGTH(af.object_class) <> 3
```

Comment:
We were only looking for '000' so we want to keep looking for that along with any permutation of it. In this case, we want to use

```
AND (op.object_class = '000' OR op.object_class = '00' OR op.object_class = '0')
```

only formatted nicer